src/components/LoginButton.jsx

```diff
@@ -0,0 +1,23 @@
+import React, {useState} from 'react';
+import './LoginButton.css'
+
+const LoginButton = () =>{
+    const [login, setLogin] = useState('Login',);
+    const [loginMessage, setLoginMessage] = useState('User is logged out.');
```

Comment:
I saw you are using prevState. Nice. I think you are overcomplicating this. We dont need link 6 useState. we could just manage it all with one variable like this:

```

  const [user, setUser] = useState(false);
  return (
    <div className="App">
      <h1>You are {user ? "Logged in" : "Logged out"}</h1>
      <button onClick={() => setUser(!user)}>
        {user ? "Log out" : "Log in"}
      </button>
    </div>
    )
```